media-libs/libdc1394/libdc1394-2.2.5-r1.ebuild

```diff
@@ -30,15 +30,17 @@ src_prepare() {
 }
 
 multilib_src_configure() {
-	local myconf="$(use_enable doc doxygen-html)"
-	multilib_is_native_abi || myconf="--disable-doxygen-html --disable-examples"
-
-	# X is only useful for examples that are not installed.
-	ECONF_SOURCE="${S}" econf \
-		$(use_enable static-libs static) \
-		--program-suffix=2 \
-		--without-x \
-		${myconf}
+	local myeconfargs=(
+		$(use_enable doc doxygen-html)
+		$(use_enable static-libs static)
+		--program-suffix=2
+		--without-x
+	)
+	# ^ X is only useful for examples that are not installed.
```

Comment:
Comments are allowed inside declarations of arrays, so this comment can be directly above `--without-x` line (and with `# ^ X` changed into `# X`).